packages/@tinacms/schema-tools/src/types/index.ts

```diff
@@ -214,8 +214,20 @@ export type ImageField = (
   }
 
 export type ReferenceField = (
-  | FieldGeneric<string, undefined>
-  | FieldGeneric<string, false>
+  | FieldGeneric<
+      string,
+      undefined,
+      { selectComponent: (props: any) => Element | undefined }
```

Comment:
How do we feel about renaming this from `select` to `option` to make it more accurate to it's implementation as it is inserted inside a `select` element

Reply:
I think this was from the pseudo code I'd written, both of these types should be (https://github.com/tinacms/tinacms/pull/4732/files#diff-c1b9d8ea97eb7c96298291e7c8940e58d0b8f56cac01609406b23872e943e935R201))

```ts
      {
        selectComponent?: (props: {
          values: Record<string, unknown>
          _sys: Document['_sys']
        }) => any
      }
```
(if you change the code [over here](https://github.com/tinacms/tinacms/pull/4732/files#diff-c1b9d8ea97eb7c96298291e7c8940e58d0b8f56cac01609406b23872e943e935R201))

Reply:
Done renamed to option